core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/QualificationToolBenchmark.scala

```diff
@@ -27,23 +27,28 @@ import com.nvidia.spark.rapids.tool.qualification.QualificationMain.mainInternal
  * 2. Write the benchmark code in the runBenchmark method passing relevant arguments
  * 3. Write benchmarked code inside
  */
-object QualificationBenchmark extends BenchmarkBase {
+object QualificationToolBenchmark extends BenchmarkBase {
```

Comment:
we can call this SingleThreadedQualTool and set the number of threads to be 1.
In tht case, when someone is running the benchmark with multiple eventlogs, it will be forced to be single threaded.

Reply:
Changed